tests/config/__init__.py

```diff
@@ -2,70 +2,88 @@
 Test GEF configuration parameters.
 """
 
-from tests.utils import gdb_run_cmd, gdb_start_silent_cmd
-from tests.utils import GefUnitTestGeneric
+from typing import List
 
+import pytest
+from tests.base import RemoteGefUnitTestGeneric
+from tests.utils import GEF_RIGHT_ARROW
 
-class TestGefConfigUnit(GefUnitTestGeneric):
-    """Test GEF configuration parameters."""
 
+class TestGefConfigUnit(RemoteGefUnitTestGeneric):
+    """Test GEF configuration parameters."""
 
     def test_config_show_opcodes_size(self):
         """Check opcodes are correctly shown."""
-        res = gdb_run_cmd("entry-break", before=["gef config context.show_opcodes_size 4"])
-        self.assertNoException(res)
-        self.assertGreater(len(res.splitlines()), 1)
+        gdb = self._gdb
+
+        max_opcode_len = 4
+        #
+        # Only show the code pane, limit the opcode size to 4 bytes (8 chars)
+        #
+        gdb.execute("gef config context.layout code")
+        gdb.execute(f"gef config context.show_opcodes_size {max_opcode_len}")
 
-        # output format: 0xaddress   opcode  <symbol+offset>   mnemo  [operands, ...]
+        gdb.execute("start")
+        lines: List[str] = gdb.execute("context", to_string=True).splitlines()[1:-1]
+        self.assertGreater(len(lines), 1)
```

Comment:
I think with our pytest style testing we should prefer `assert len(lines) > 1`

Reply:
> I think with our pytest style testing we should prefer `assert len(lines) > 1`

Correct I intend to do a pass of `sed` right after to replace all those `self.assert*` to `assert`
But since it's not really urgent right now the focus was on just porting the tests as close to what they were to facilate review.